### PR TITLE
New pipes: ARQ based on rfc4585

### PR DESCRIPTION
**Pull request by funman**

### CODE REVIEW

File: lib/upipe-modules/upipe_rtcp_fb_receiver.c [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
```
Comment (cmassiot) [inline lib/upipe-modules/upipe_rtcp_fb_receiver.c]: If you want LGPL you cannot put it in upipe-modules

File: lib/upipe-modules/upipe_rtp_feedback.c [hunk 2]
```diff
@@ -0,0 +1,1034 @@
...
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
```
Comment (cmassiot) [inline lib/upipe-modules/upipe_rtp_feedback.c]: Same here.

File: lib/upipe-modules/upipe_rtcp_fb_receiver.c (hunk 1 above)
Comment (funman) [reply lib/upipe-modules/upipe_rtcp_fb_receiver.c]: @kierank

**Comment by cmassiot** [comment]
Can you squash it ?

**Pull request merged**